docs/data/toolpad/concepts/custom-components.md

```diff
@@ -4,6 +4,17 @@
 
 ## `createComponent`
 
+Creating a custom component in Toolpad is a quick process:
```

Comment:
Let's avoid language that describes steps as quick/easy/simple? (https://www.notion.so/Writing-style-guide-2a957a4168a54d47b14bae026d06a24b?d=bf8ae75e0eb94c96b6328a61367eb191&pvs=4#ad0d7e5fd0e6494a97f6febff143af96)